site/static/otel-collector.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector-config
+data:
+  otel-collector-config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:4317
+    processors:
+    extensions:
+    exporters:
+      otlp/jaeger:
+        endpoint: "jaeger.tracing.svc:4317"
+        tls:
+          insecure: true
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: []
+          exporters: [otlp/jaeger]
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: otel-collector
+  labels:
+    app.kubernetes.io/name: otel-collector
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: otel-collector
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: otel-collector
+    spec:
+      containers:
+      - name: otel-collector
+        image: otel/opentelemetry-collector:latest
```

Comment:
I wonder if using latest images both for jagger and collector could create problems in the future. For example, the current pod spec will not work for the future image version.

Reply:
Yeah I go back and forth on this. The issue with pinning it is that it will become outdated at some point.

Reply:
I wonder if we need to create a periodic issue or something to ensure examples run without errors?

Reply:
what if the automation assigns it and adds to the current sprint? 😆

Reply:
I think that's asking for trouble haha